### PR TITLE
[8.0][website_snippet_contact_form] Contact form in a snippet

### PR DESCRIPTION
**Pull request by yajo**
CC @rafaelbn @antespi.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Better to call it _website_snippet_crm_

File: website_crm_snippet/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Contact Form Snippet",
+    "summary": "Put a contact form anywhere you want",
+    "version": "8.0.1.0.0",
+    "category": "Uncategorized",
```
Comment (pedrobaeza) [inline website_crm_snippet/__openerp__.py]: Put the same category as Website

**Comment by pedrobaeza** [comment]
For the icon, please follow the same style as https://github.com/odoo/odoo/tree/8.0/addons/website/static/src/img/blocks

**Comment by rafaelbn** [comment]
> Better to call it website_snippet_crm

Odoo core module which add contact form is website_crm, this module adds a snippet, don't you think @pedrobaeza website_crm_snippet is more correct an intuitive?

**Comment by pedrobaeza** [comment]
No, but the standard way to name this is website_snippet__, so that you have all the snippets together sorting alphabetically. It's the same as the widgets for backend: web_widget__.

**Comment by rafaelbn** [comment]
Tested, please:
- You don't show checkbox for accepting privacy policy but form expect it


- Improve a little bit design, make fields not full width



Thanks

**Comment by pedrobaeza** [comment]
You should add the possibility to add the form as a floating element, and maybe this can replace what @rafaelbn says about the size, because the width will adapt to the available space in the upper container.

**Comment by yajo** [comment]
The point of _crm_ in name is to agree with _website_crm_ module name. If I prefix this with _website_snippet_, I'll call it _website_snippet_contact_form_. About the other problems, I'm on it.

**Comment by pedrobaeza** [comment]
OK, I agree with your suggested name.

**Comment by rafaelbn** [comment]
I agree

**Comment by yajo** [comment]
I'm trying to make it able to be resized as ssuggested, but no way! I always get this error in JS console: `have not insert action: data-drop-near or data-drop-in`, but no idea on where it comes from or how to fix! Can you help please?

**Comment by rafaelbn** [comment]
@dgzurita could you review and help please?

**Comment by yajo** [comment]
Still lacking an appropiate icon for the snippet, but that does not seem very critical for version 1.0, right?

**Comment by rafaelbn** [comment]
Where is runbot?

**Comment by antespi** [comment]
:+1: tested in runbot

**Comment by rafaelbn** [comment]
Tested and :+1: great job!

bd231872-6845-11e5-8550-1f071c9e5e5c

**Comment by dgzurita** [comment]
:+1: tested in runbot

**Comment by pedrobaeza** [comment]
What about the possibility of configuring the fields to show?

**Comment by rafaelbn** [comment]
It takes the parameters of the standard contact form. We will see to improve it.

**Comment by pedrobaeza** [comment]
You should mention then on the Known issues section. Although the icon part.

**Pull request merged**